tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh

```diff
@@ -32,7 +32,26 @@ for dir in $DIRS
 do
   for glob in $GLOB
   do
-    files="$files `find ${CLANG_FORMAT_ROOT}/$dir -name $glob -and -not -name '*.generated.*' -and -not -name '*.upb.h' -and -not -name '*.upb.c' -and -not -name '*.upbdefs.h' -and -not -name '*.upbdefs.c' -and -not -name '*.pb.h' -and -not -name '*.pb.c' -and -not -name '*.pb.cc' -and -not -name '*.pbobjc.h' -and -not -name '*.pbobjc.m' -and -not -name '*.pbrpc.h' -and -not -name '*.pbrpc.m' -and -not -name end2end_tests.cc -and -not -name grpc_shadow_boringssl.h -and -not -name grpc_tls_credentials_options.h -and -not -name grpc_tls_credentials_options_comparator_test.cc`"
+    files="$files `find ${CLANG_FORMAT_ROOT}/$dir -name $glob \
+    -and -not -name '*.generated.*' \
+    -and -not -name '*.upb.h' \
+    -and -not -name '*.upb.c' \
+    -and -not -name '*.upbdefs.h' \
+    -and -not -name '*.upbdefs.c' \
+    -and -not -name '*.pb.h' \
+    -and -not -name '*.pb.c' \
+    -and -not -name '*.pb.cc' \
+    -and -not -name '*.pbobjc.h' \
+    -and -not -name '*.pbobjc.m' \
+    -and -not -name '*.pbrpc.h' \
+    -and -not -name '*.pbrpc.m' \
+    -and -not -name end2end_tests.cc \
+    -and -not -name grpc_shadow_boringssl.h \
+    -and -not -name grpc_tls_credentials_options.h \
+    -and -not -name grpc_tls_credentials_options_comparator_test.cc \
+    -and -not -path '*/cmake/build/*' \
```

Comment:
I'm curious why cmake files happen to reach here. Do we have cmake files under any directory from `DIRS`?

Reply:
It depends on the state of your local environment, and I suppose the state of the CI build environment. I didn't go out of my way to run anything cmake-related, but I get 5901 entries for `find . -type d -path '*/cmake/build/*' | wc -l`

Reply:
```
$ find examples/cpp -type d -path '*/cmake/build/*' | wc -l
1154
```

Reply:
oh i see. examples! sg